shared/constants/selectors.js

```diff
@@ -5,10 +5,19 @@ import type {SearchQuery} from './searchv3'
 const usernameSelector = ({config: {username}}: TypedState) => username
 const loggedInSelector = ({config: {loggedIn}}: TypedState) => loggedIn
 
-// $FlowIssue with accessing records
 const cachedSearchResults = ({entities: {searchQueryToResult}}: TypedState, searchQuery: SearchQuery) =>
   searchQueryToResult.get(searchQuery)
 
 const inboxSearchSelector = ({chat: {inboxSearch}}: TypedState) => inboxSearch
 
-export {cachedSearchResults, inboxSearchSelector, loggedInSelector, usernameSelector}
+const amIFollowing = ({config: {following}}: TypedState, otherUser: string) => following[otherUser]
```

Comment:
Should these be converted to booleans? Will this selector return `undefined` if not following a user?